src/scripts/pages/login/main.js

```diff
@@ -0,0 +1,26 @@
+import Cookies from 'js-cookie';
+import { auth, FacebookAuthProvider, signInWithPopup } from '../../../services/firebase';
+
+Cookies.set('foo', 'bar');
+const provider = new FacebookAuthProvider();
+const facebookLoginButton = document.getElementById('login-facebook-button');
+
+facebookLoginButton.addEventListener('click', async function () {
+    const data = await signInWithPopup(auth, provider);
+    handleFBLogin(data);
+});
+
+function handleFBLogin(data) {
+    // Clear cookie
+    Cookies.remove('DataFromFb');
+    Cookies.remove('accessTokenFromFb');
+    //set cookie
+    Cookies.set('DataFromFb', JSON.stringify(data.user), { expires: 7 });
```

Comment:
what does it mean { expires: 7 }

Reply:
it means that access_token is only valid for 7 days